tests/AddressBookGUITest.java

```diff
@@ -134,6 +164,31 @@ public void cantCreateBadNewPerson() {
         window.table().requireRowCount(0);
     }
 
+    /**
+     * This is a negative test to check behavior of when
+     * the GUI tries to add an invalid person.
+     * Invalid last name as well.
+     */
+    @Test
+    public void cantCreateBadNewPersonLastName() {
+        // Click and get dialog window
+        window.button("add").click();
+        DialogFixture dialog = window.dialog();
+
+        // Type just 'John','Doe'
+        dialog.textBox("firstName").enterText("John");
+
+        // Click 'OK'
+        dialog.button(JButtonMatcher.withText("OK")).click();
+
+        // Test that there are still no people
+        window.table().requireRowCount(0);
+    }
```

Comment:
That's weird, why was my function not in the code you are merging too??

Reply:
Is there a missing function? I made sure to pull before pushing

Reply:
No, this for some reason shows it is adding a function that doesn't seem to exist. But it is a function I created so it is weird to show it not existing. Probably doesn't matter, I will mark this as resolved